tests/cpp-runtime/hexagon/hexagon_thread_manager_tests.cc

```diff
@@ -161,7 +162,8 @@ TEST_F(HexagonThreadManagerTest, pipe_fill) {
   CHECK_EQ(answer, 42);
 }
 
-TEST_F(HexagonThreadManagerTest, pipe_overflow) {
+// TODO(HWE): Create a temporary thread manager with a smaller pipe for this test
```

Comment:
Per discussion this morning, one way around this is to split out a base class for thread management, and then a HexagonThreadManager that inherits this.  Hardware acquisition only occurs from the HexagonThreadManager.  We could use that base class to test pipe length, etc.

Reply:
Yes, would like to know the feasibility of splitting into testable base class + HexagonThreadManager before we decide whether to land this PR with this disabled test.

Reply:
I think this might have some complexity when we parameterize the thread_main for each hardware resource that will be dedicated to that thread.  So I'd like to postpone this until that task to acquire one hardware resource per thread.